RiotSharp/RiotSharp.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$title$</title>
+    <authors>$author$</authors>
+    <owners>$author$</owners>
+    <licenseUrl>https://github.com/BenFradet/RiotSharp</licenseUrl>
+    <projectUrl>https://github.com/BenFradet/RiotSharp/blob/develop/LICENSE.md</projectUrl>
```

Comment:
I think those two are inverted, `licenseUrl` should be `projectUrl` and vice-versa.